docs/blog/2021-12-09-log4j-zero-day.mdx

```diff
@@ -268,25 +269,25 @@ import ContactForm from '../src/components/ContactForm.jsx'
 
 ### Edits
 
-1. Updated the "Who is impacted?" section to include mitigating factor based on JDK version, but also suggest other exploitation
-methods are still prevalent.
-2. ~~Named the vulnerability "LogJam",~~ added CVE, and added link to release tags.
-3. Update mitigation steps with newer information.
+1. Updated the "Who is impacted?" section to include mitigating factors based on JDK version, while also suggesting other exploitation
+methods as still prevalent.
+2. ~~Named the vulnerability "LogJam"~~ Added CVE, and added link to release tags.
+3. Updated mitigation steps with newer information.
 4. Removed the name "LogJam" because it's already been [used](https://en.wikipedia.org/wiki/Logjam_(computer_security)). Using "Log4Shell" instead.
 5. Update that 2.15.0 is released.
 6. Added the MS Paint logo[4], and updated example code to be slightly more clear (it's not string concatenation).
-7. Reported on iPhones being affected by the vulnerability, and included local reproduction code + steps.
-8. Update social info.
+7. Reported on iPhones being affected by the vulnerability, and included local reproduction code and steps.
+8. Updated social info.
 9. Updated example code to use Log4j2 syntax.
-10. Update title because of some confusion.
+10. Updated title because of some confusion.
 11. Better DNS testing site and explanation
 12. Added link to the [Log4Shell Mitigation Guide](https://www.lunasec.io/docs/blog/log4j-zero-day-mitigation-guide/).
 13. Add warnings about limited vuln in 2.15 / noMsgFormatLookups
 14. Added link to 2nd CVE.
 15. Updated contact information.
-16. Updated original twitter link from @P0rZ9 as the original tweet was deleted. Changed from `https://twitter.com/P0rZ9/status/1468949890571337731` to `https://web.archive.org/web/20211209230040/https://twitter.com/P0rZ9/status/1468949890571337731`
+16. Updated original Twitter link from @P0rZ9 as the original tweet was deleted. Changed from `https://twitter.com/P0rZ9/status/1468949890571337731` to `https://web.archive.org/web/20211209230040/https://twitter.com/P0rZ9/status/1468949890571337731`.
 17. Added links to other blog posts.
-18. Update post to include latest version 2.17.0 release.
+18. Updated post to include latest version 2.17.0 release.
```

Comment:
Nice changing to past tense.  You can see the developer in us coming out because commit messages are present tense like this :P

Reply:
Yea just consistency.